### PR TITLE
Fixes poll buttons and unrounded images button

### PR DESCRIPTION
**Pull request by DylanPlemmons**
Fixed poll buttons and the unrounded image button (credit to @PandawanFr for fixing the unrounded image button).

### CODE REVIEW

**Comment by Pandawan** [comment]
Thanks, I was kinda lazy to create a PR.

**Comment by Pandawan** [comment]
It seems you didn't add the weird shadow at the bottom of the search bar's suggestions.

File: darkmode.scss [hunk 1]
```diff
@@ -548,6 +548,11 @@
     }
   }
 
+  .tweet-form.condensed * .btn {
```
Comment (pugson) [inline darkmode.scss]: @DylanPlemmons The `*` selector is pretty expensive on the browser. We're applying a whole stylesheet on top of Twitter's stylesheet, so there's already a slight performance hit. Is there any other way you could make this code work without using the `*` selector?

**Comment by pugson** [comment]
Thanks @DylanPlemmons and @PandawanFr 👍 💯

**Pull request merged**